client/my-sites/migrate/section-migrate.jsx

```diff
@@ -670,12 +671,22 @@ export class SectionMigrate extends Component {
 	}
 
 	render() {
-		const { step, sourceSite, targetSite, targetSiteSlug, translate } = this.props;
+		const { step, sourceSite, targetSite, targetSiteSlug, translate, targetSiteId } = this.props;
 		const sourceSiteSlug = get( sourceSite, 'slug' );
 
 		let migrationElement;
 
 		switch ( this.state.migrationStatus ) {
+			case 'in-progress':
+				return (
+					<MigrationInProgress
+						sourceSite={ get( sourceSite, 'domain' ) }
+						targetSite={ get( targetSite, 'domain' ) }
```

Comment:
Nit: We want to move away from using `lodash`, so it would be better if we avoid calling `get()`.
```suggestion
						sourceSite={ sourceSite?.domain }
						targetSite={ targetSite?.domain }
```

Reply:
Fixed https://github.com/Automattic/wp-calypso/pull/88916/commits/c14943e48d2e703c890edbdc842371ca88c3f640